packages/meditrak-server/src/s3/S3Client.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import AWS from 'aws-sdk';
+import { BUCKET_NAME } from './constants';
+
+export class S3Client {
+  constructor() {
+    this.s3 = new AWS.S3();
+  }
+
+  async checkIfFileExists(fileName) {
+    return new Promise(resolve => {
+      this.s3
+        .headObject({
+          Bucket: BUCKET_NAME,
+          Key: fileName,
+        })
+        .on('success', () => {
+          resolve(true);
+        })
+        .on('error', () => {
+          resolve(false);
```

Comment:
We could check that the error thrown is specifically because of the file not found, and not for another reason

Reply:
True, that would be a more cautious approach!

Reply:
I was just about to go and do that (because I think it's a good idea) and then I remembered how we have to cut a few corners if we hope to deliver psss by 11th Dec, so I'm going to ignore your suggestion - sorry!

Reply:
No worries - I think it's looking great as is!